package.json

```diff
@@ -32,8 +32,8 @@
     "test": "yarn test:ng",
     "test:smoke": "sh whitelist.sh",
     "test:smoke:local": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/smoke.conf.js --local",
-    "test:functional:backup": "echo 'Not implemented'",
-    "test:functional": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/conf.js",
+    "test:functional": "echo 'Not implemented'",
+    "test:functional:backup": "webdriver-manager update --versions.chrome 2.40 && protractor ./test/e2e/config/conf.js",
```

Comment:
undo these changes as it's effectively disabling functional tests?

Reply:
Done @paddybasi